### ISSUE TITLE
integrate with splunk

### ISSUE THREAD

**Issue by alirezakatani** [GACWR/OpenUBA#99]
hello. is your application have ability to integrate with splunk or not?

**Comment by Jovonni** [comment]
Yes — Splunk integration is up in #145. It works both ways: you can run a Splunk search (SPL) as a data source for any model, and optionally forward detected anomalies back into Splunk via HEC. Configurable under Settings → Integrations → Splunk (REST for search, HEC for output), with a connectivity test. Docs in docs/SPLUNK.md.
